iplocation/ip2location/iplocation.go

```diff
@@ -0,0 +1,61 @@
+package ip2location
```

Comment:
The thing we use to resolve location from ip addrs.
TBH, its quite simple. The client SDK is very poor in design: it uses globals and that sorts of things... but it works.

Clients rely on an interface that satisfies `Resolve(mas []multiaddr.Multiaddr) (iplocation.Location, error)`, so we can change this to other implementation or whatever easily if necessary.